.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Upload virtualenv.pyz to release
+
+on:
+  release:
+    types: [published]
+
+# Sets the permissions for GITHUB_TOKEN to allow writing to releases
+permissions:
+  contents: write
+
+jobs:
+  upload-asset:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v5
+
+      - name: Upload virtualenv.pyz to release
+        run: |
+          upload_url=${{ github.event.release.upload_url }}
+
+          # Remove the template bit at the end
+          upload_url="${upload_url%\{*\}}"
+          # add the asset name
+          upload_url="${upload_url}?name=virtualenv.pyz"
+
+          curl -L --fail-with-body \
```

Comment:
Shouldn't we also upload version specific versions too? https://github.com/pypa/get-virtualenv/tree/main/public